proposals/texture-component-swizzle.md

```diff
@@ -52,15 +40,13 @@ To reduce compatibility issues in practice, implementations *should* provide (V,
 
 ## Validation
 
-The `GPUTexture.createView(descriptor)` algorithm is extended with the following validation rules changes:
+The `GPUTexture.createView(descriptor)` algorithm is extended with the following changes:
 
-- If `descriptor.usage` includes the `RENDER_ATTACHMENT` or `STORAGE_BINDING` bit:
-  - `descriptor.swizzle.r` must be `"r"`.
-  - `descriptor.swizzle.g` must be `"g"`.
-  - `descriptor.swizzle.b` must be `"b"`.
-  - `descriptor.swizzle.a` must be `"a"`.
+- If `descriptor.swizzle` must be a four-character string that only includes `"r"`, `"g"`, `"b"`, `"a"`, `"0"`, or `"1"`, otherwise a `TypeError` is raised.
```

Comment:
```suggestion
- `descriptor.swizzle` must be a four-character string that only includes `"r"`, `"g"`, `"b"`, `"a"`, `"0"`, or `"1"`, otherwise a `TypeError` is raised.
```